helix-term/src/ui/editor.rs

```diff
@@ -666,10 +666,10 @@ impl EditorView {
         // Left side of the status line.
         //-------------------------------
 
-        let mode = match doc.mode() {
-            Mode::Insert => "INS",
-            Mode::Select => "SEL",
-            Mode::Normal => "NOR",
+        let (mode, mode_style) = match doc.mode() {
+            Mode::Insert => (" INS ", theme.get("ui.statusline.insert")),
+            Mode::Select => (" SEL ", theme.get("ui.statusline.select")),
+            Mode::Normal => (" NOR ", theme.get("ui.statusline.normal")),
```

Comment:
The new scopes will also have to be added to the book in the [Themes](https://docs.helix-editor.com/master/themes.html#interface) section.

Reply:
Oh, I didn't realize the book was in the same repo, awesome. On it!

Reply:
@Mathspy - are you still working on docs? Eager to see this merged.

Reply:
Added documentation for the new scopes in https://github.com/helix-editor/helix/pull/2676/commits/107e65caa7a265193dc4accb226358ac29cd4552